test/integration/app-proxy-manager-agent.yaml.tmpl

```diff
@@ -40,7 +40,7 @@ spec:
             "image": "{{.hub}}/init:{{.tag}}",
             "args": ["-p", "15001", "-u", "1337"],
             "imagePullPolicy": "Always",
-            "securityContext": { "capabilities" : { "add" : ["NET_ADMIN"] } }
+            "securityContext": { "capabilities" : { "add" : ["NET_ADMIN"] }, "privileged": true }
```

Comment:
Let's remote sysctl command from init container and put it into debug container. Core dumps are only useful if you are willing to debug them.